yb-voyager/src/query/queryparser/traversal_proto.go

```diff
@@ -52,6 +52,9 @@ const (
 	PG_QUERY_JSON_IS_PREDICATE_NODE          = "pg_query.JsonIsPredicate"
 	PG_QUERY_VIEWSTMT_NODE                   = "pg_query.ViewStmt"
 	PG_QUERY_COPYSTSMT_NODE                  = "pg_query.CopyStmt"
+
+	PG_QUERY_TABLECONSTRAINT_NODE = "pg_query.Constraint"
```

Comment:
lets keep the constant name same as node name - `PG_QUERY_CONSTRAINT_NODE` ?

Reply:
Yeah makes sense. Done.